Formula/checkov.rb

```diff
@@ -160,24 +165,44 @@ class Checkov < Formula
     sha256 "594c67807fb16238b30c44bdf74f36c02cdf22d1c8cda91ef8a0ed8dabf5620a"
   end
 
+  resource "matplotlib" do
+    url "https://files.pythonhosted.org/packages/60/d3/286925802edaeb0b8834425ad97c9564ff679eb4208a184533969aa5fc29/matplotlib-3.4.2.tar.gz"
+    sha256 "d8d994cefdff9aaba45166eb3de4f5211adb4accac85cbf97137e98f26ea0219"
+  end
+
   resource "networkx" do
-    url "https://files.pythonhosted.org/packages/b0/21/adfbf6168631e28577e4af9eb9f26d75fe72b2bb1d33762a5f2c425e6c2a/networkx-2.5.1.tar.gz"
-    sha256 "109cd585cac41297f71103c3c42ac6ef7379f29788eb54cb751be5a663bb235a"
+    url "https://files.pythonhosted.org/packages/38/d4/92b512ba330a8852f5f6bf059917d2bcaf7525b3e40853a2d826cfbda200/networkx-2.6.1.tar.gz"
+    sha256 "bf4cb807d1bccf1593c7d0742d9127d9e04e021867299082658b0fc3907924e8"
+  end
```

Comment:
The build failure here is similar to the `networkx` issue seen inc #80829, we should wait until 2.7 to avoid the circular dependency. It might be simple to update resources other than `networkx` for now.

Reply:
@chenrui333 We'll also have to remove the resources that were added as a result of the `networkx` bump, like `numpy`, `pandas`, `scipy`, etc.